test/CMakeLists.txt

```diff
@@ -362,6 +362,45 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "Linux" AND CMAKE_CXX_COMPILER_ID STREQUAL "GNU
     )
 endif()
 
+# Only produce timing tests for UNIX based systems (faketime constraint)
```

Comment:
Could you add a note about how this test works (we link libfaketime, and set the environment override at the bottom so that it applies).

Reply:
Changed to LD_PRELOAD